src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.Sdk.props

```diff
@@ -81,6 +81,8 @@ Copyright (c) .NET Foundation. All rights reserved.
     <DelaySign Condition="'$(DelaySign)'==''">false</DelaySign>
     <GeneratePackageOnBuild Condition="'$(GeneratePackageOnBuild)'==''">false</GeneratePackageOnBuild>
     <PackageRequireLicenseAcceptance Condition="'$(PackageRequireLicenseAcceptance)'==''">false</PackageRequireLicenseAcceptance>
+    <DebugSymbols Condition="'$(DebugSymbols)'==''">false</DebugSymbols>
```

Comment:
Is there a way to know that this is the exhaustive list? Can you make sure to play with each property with the CPS Visible=False to make sure that it doesn't crash?